walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdRequest.kt

```diff
@@ -14,16 +14,7 @@ import com.microsoft.walletlibrary.util.VerifiedIdResult
  * requirements needed in order to complete a request and information about trust model of requester
  * like domain url and verification status.
  */
-interface VerifiedIdRequest<out T> {
-    // Attributes describing the requester (eg. name, logo).
-    val requesterStyle: RequesterStyle
```

Comment:
One of the key differences between the verifiedIdPartialRequest and the VerifiedIdRequest is the mutability of these fields. In mobile app code, none of these parameters should be mutable. IE, the developer does not believe that they can simply remove or replace the requirement to satisfy it.